src/components/badges/List.tsx

```diff
@@ -55,31 +45,31 @@ function BadgeListSuspended() {
   ) : isEmpty ? (
     <HintCard text="You don't own any derivatives and you don't have any ZK proofs ready to use. Generate a ZK proof first!" />
   ) : (
-    <div className={space('space-y-2')}>
-      <BadgeSection
-        title="NFT derivatives"
-        minted={ownedERC721DerivativeContracts}
-        proofs={proofsAvailableToMint.filter(
-          (proof) => proof instanceof ERC721Proof
-        )}
-      />
-      <BadgeSection
-        title="Email derivatives"
-        minted={ownedEmailDerivativeContracts}
-        proofs={proofsAvailableToMint.filter(
-          (proof) => proof instanceof EmailProof
-        )}
-      />
-    </div>
+    <Scrollbar>
```

Comment:
We shouldn't use `Scrollbar` outside the really scrollable area, it will cause problems, like the `<HintCard />` above was scrollable